lametro/__init__.py

```diff
@@ -121,3 +121,9 @@ def get_organizations(self):
             "https://metro.legistar.com/DepartmentDetail.aspx?ID=30222&GUID=5F27DA83-633F-4FEA-A4B0-0477551061B6&R=aef57793-1826-4cfa-b6e3-d6b42cf77527"
         )
         yield org
+
+        org = Organization(name="Special Board Budget Workshop", classification="legislature")
```

Comment:
I used the same classification as the Board because `Special Board Budget Workshop` and `Board of Directors - Regular Board Meeting` both have a `BodyTypeName` of `Primary Legislative Body`.

Reply:
Is Special Board Budget Workshop a child of the board, or a sibling?

Reply:
In the `ArrayofGranicusBody` structure in [Legistar](https://webapi.legistar.com/v1/metro/bodies), Special Board Budget Workshop is a sibling of the regular board meeting.